kolibri/plugins/learn/assets/src/utils/genContentLink.js

```diff
@@ -1,8 +1,10 @@
 import { PageNames } from '../constants';
 
-export default function genContentLink(id, isLeaf) {
+// previous context allows to pass in ids or params from the last route
+export default function genContentLink(id, isLeaf, last, prevContext) {
   return {
     name: isLeaf ? PageNames.TOPICS_CONTENT : PageNames.TOPICS_TOPIC,
     params: { id },
+    query: { last: last, prevContext },
```

Comment:
How does the `prevContext` object get encoded into the URL? I assume it's fine?

Reply:
That's an oversight -- I've now added encoding